students/Lincoln_Zhang/Session6/test_mailroom_part4.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import mailroom_part4, os
+import unittest.mock
+
+def test_gen_letter():
+    result = mailroom_part4.gen_letter("Bob", 11.2)
+    assert result.startswith("\nDear Bob")
+    assert result.find("11.2")
```

Comment:
careful here! .find() returns the index of what it's found. and as zero is a valid index, it used -1 to mean "not found" -- but -1 is "truthy" -- so this assertion will always pass!

`assert "11.2" in result`

is a better check -- better still, something like:

`assert "$11.20" in result`

so you can see that it was formatted correctly.